website/docs/docs/cloud/git/connect-github.md

```diff
@@ -77,7 +76,7 @@ To connect a personal GitHub account:
 
 4. Once you approve authorization, you will be redirected to dbt Cloud, and you should now see your connected account. 
 
-The next time you log into dbt Cloud, you will be able to do so via OAuth through GitHub, and if you're on the Enterprise plan, you're ready to use the dbt Cloud IDE or dbt Cloud CLI.
+You're now ready to use the dbt Cloud IDE or dbt Cloud CLI.
```

Comment:
```suggestion
You can now use the dbt Cloud IDE or dbt Cloud CLI.
```

Reply:
does it make sense to link out to the IDE and CLI pages here?

Reply:
Linked at the top of this section!